css/style.css

```diff
@@ -0,0 +1,798 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap');
```

Comment:
I see you're importing your fonts in 2 different ways: with @import here and also in your HTML `<head>`. You only need one of these (probably stick to `head` for now)

The fonts for this project should be different as described in the readme - can you change them? It will make a world of difference to how your site looks 🙂